### DISCUSSION TITLE
Chart not displayed when panning on Y axis

### DISCUSSION THREAD

**Discussion by electronicbits** [reactivemarkets/react-financial-charts#695] [Q&A]
As shown in the video below, when panning/dragging on Y axis, the chart is not displayed, only at panend event execution. 

LIkely issue [#692](https://github.com/react-financial/react-financial-charts/issues/692) is referring to this (?)

However, it hasn't received any reply, so decided to open it for discussion first.

Have debugged the code and inspected the ChartCanvas drawing operations during panning but can't see anything wrong so far.

Has anyone come around a fix or workaround for this ? 

![issue-charts](https://github.com/react-financial/react-financial-charts/assets/1396353/41b3906f-dd83-4e6a-b60c-a3517108438a)

**Comment by dnjsgur0629** [comment]
Sorry this commnet is not a solution for the chart not displaying.
I now need y-axis panning. Can you tell me how you achieved y-axis panning?

**Reply by electronicbits** [reply]
Rollback to the previous working version, then apply all changes manually, one by one. 
It was a bit time consuming, and I still don't know exactly where/what was causing this issue, but I managed to solve it by doing what I just said. The added benefit is that created a bit more awareness of the code base and changes, and how the code is architected (or not :) )
